app/src/features/apiClient/screens/apiClient/components/modals/deleteApiRecordModal/DeleteApiRecordModal.tsx

```diff
@@ -13,6 +13,10 @@ import { useApiClientContext } from "features/apiClient/contexts";
 import { trackCollectionDeleted } from "modules/analytics/events/features/apiClient";
 import "./deleteApiRecordModal.scss";
 
+enum RecordType {
+  API = "api",
+  COLLECTION = "collection",
+}
```

Comment:
No need to create a new enum. import the `RQAPI` namespace and then use the `RQAPI.RecordType` in recordType checks